tests/Application/src/Tests/Controller/SubscriptionXmlApiTest.php

```diff
@@ -34,89 +40,235 @@ public function it_allows_showing_a_subscription(): void
             ->create()
         ;
 
-        $this->client->request('GET', '/ajax/subscriptions/' . $subscription->getId());
-        $response = $this->client->getResponse();
+        $this->client->request('GET', '/ajax/subscriptions/' . $subscription->getId(), server: self::$headersWithContentType);
+
+        $this->assertResponseIsSuccessful();
+        $this->assertResponseStatusCodeSame(Response::HTTP_OK);
+        $this->assertResponseHeaderSame('content-type', 'text/xml; charset=utf-8');
```

Comment:
Same question, do we need charset check over here?

```suggestion
        $this->assertResponseHeaderSame('content-type', 'text/xml');
```